### PR TITLE
Added sync temperature call method 

### PR DESCRIPTION
**Pull request by kkoehler**
HI,

i've added a sync temperature call method and a sample how it could be called.

Kristian

### CODE REVIEW

File: lib/ds18b20.js [hunk 1]
```diff
@@ -28,11 +28,31 @@ var sensors = function(callback) {
...
+    
+    if (arr[1].charAt(0) === 'f') {
+        
+        var x = parseInt("0xffff" + arr[1].toString() + arr[0].toString(), 16);
+        var output = -((~x + 1) * 0.0625);
+        return (-((~x + 1) * 0.0625));
+        
+    } else if (arr[1].charAt(0) === '0') {
+        
+        var output = parseInt("0x0000" + arr[1].toString() + arr[0].toString(), 16) * 0.0625;
```
Comment (chamerling) [inline lib/ds18b20.js]: please return output directly instead of creating the variable

File: samples/index.js [hunk 2]
```diff
@@ -21,4 +21,16 @@ sensor.sensors(function(err, ids) {
...
+});
+
+sensor.sensors(function(err, ids) {
+  if (err) {
+    console.log('Can not get sensor IDs', err);
+  } else {
+    console.log(ids);
+    for (var id in ids) {
+        console.log('Sensor ' + ids[id] + ' :' + sensor.temperatureSync(ids[id]));
+      ;
```
Comment (chamerling) [inline samples/index.js]: remove this line

**Comment by kkoehler** [comment]
changed code according to your comments

**Comment by chamerling** [comment]
@kkoehler merged, thx, will push a new version on npm asap;

**Pull request merged**